website/docs/r/fsx_lustre_file_system.html.markdown

```diff
@@ -36,6 +36,8 @@ The following arguments are supported:
 * `kms_key_id` - (Optional) ARN for the KMS Key to encrypt the file system at rest, applicable for `PERSISTENT_1`. Defaults to an AWS managed KMS Key.
 * `per_unit_storage_throughput` - (Optional) - Describes the amount of read and write throughput for each 1 tebibyte of storage, in MB/s/TiB, required for the `PERSISTENT_1` deployment_type. For valid values, see the [AWS documentation](https://docs.aws.amazon.com/fsx/latest/APIReference/API_CreateFileSystemLustreConfiguration.html).
 * `automatic_backup_retention_days` - (Optional) The number of days to retain automatic backups. Setting this to 0 disables automatic backups. You can retain automatic backups for a maximum of 35 days. only valid for `PERSISTENT_1` deployment_type.
+* `storage_type` - (Optional) - The filesystem storage type. Either `SSD` or `HDD`, defaults to `SSD`.
+* `drive_cache_type` - (Optional) - Required for `HDD` storage_type, whether to use an SSD drive cache, either `READ` or `NONE`.
```

Comment:
Let's add some more info here: start with `The type of drive cache used by PERSISTENT_1 file systems that are provisioned with HDD storage devices.`

Reply:
done, added your sentence, and modified the end of it.